protocol/avdl/chat1/chat_ui.avdl

```diff
@@ -104,6 +104,8 @@ protocol chatUi {
   record UIMessageValid {
     MessageID messageID;
     gregor1.Time ctime;
+    gregor1.Time now;
```

Comment:
tweak this to send computed values for ephemeral stuff (isExpired, isHidden etc)

Reply:
done